solver/src/liquidity/offchain_orderbook.rs

```diff
@@ -20,18 +20,18 @@ impl OrderBookApi {
     }
 }
 
-impl Into<LimitOrder> for OrderCreation {
-    fn into(self) -> LimitOrder {
+impl From<OrderCreation> for LimitOrder {
+    fn from(order: OrderCreation) -> Self {
```

Comment:
What was this issue here? Is Into no longer a thing?

Reply:
This is the warning I got from clippy https://rust-lang.github.io/rust-clippy/master/index.html#from_over_into .